independent-projects/tools/message-writer/src/main/java/io/quarkus/devtools/messagewriter/MessageIcons.java

```diff
@@ -1,14 +1,29 @@
 package io.quarkus.devtools.messagewriter;
 
-public final class MessageIcons {
+import io.smallrye.common.os.OS;
 
-    public static final String OK_ICON = "\u2705";
-    public static final String NOK_ICON = "\u274c";
-    public static final String NOOP_ICON = "\uD83D\uDC4D";
-    public static final String WARN_ICON = "\uD83D\uDD25";
-    public static final String ERROR_ICON = "\u2757";
+public enum MessageIcons {
 
-    private MessageIcons() {
+    OK_ICON("\u2705", "[OK]"),
+    NOK_ICON("\u274c", "[NOK]"),
+    NOOP_ICON("\uD83D\uDC4D", "[NOOP]"),
```

Comment:
```suggestion
    NOOP_ICON("\uD83D\uDC4D", ":)"),
```

Reply:
If using smiles, use `:(` for NOK and `;)` for OK 😉

Reply:
Please, provide a list with the values you want for each case.

Reply:
Seriously, I don't think we should use smiley for that. It's really unclear.

Reply:
And also it's harder to parse if you want to look for errors. Let it be `[SUCCESS]` `[FAILURE]` as in Maven. That would be a lot clearer and easier to manipulate if needed.

Reply:
Let me see if I got this straight. @gsmet So you prefer sending `[SUCCESS]` `[FAILURE]` `[WARNING]` `[ERROR]` instead of all these icons, right?